### ISSUE TITLE
ARIA role: dropdown incorrect

### ISSUE THREAD

**Issue by iantrudell** [ng-select/ng-select#403]
role: "dropdown" throws an Accessibility error in Chrome's Lighthouse audits, perhaps it should be role: "combobox" instead?

In ng-select.js, ng-select.umd.js, ng-select.umd.min.js

### Expected behaviour
host: {
        'role': 'combobx',
        'class': 'ng-select',
        '[class.ng-single]': '!multiple',
    }

### Actual behaviour
host: {
        'role': 'dropdown',
        'class': 'ng-select',
        '[class.ng-single]': '!multiple',
    }

**Comment by esoyke** [comment]
Why was this closed? I see the same error reported from the aXe dev tool, 'dropdown' isn't a valid role.
